### ISSUE TITLE
Sample Size Needed to Compare k Proportions

### ISSUE THREAD

**Issue by michaelamoako** [fairlearn/fairlearn#796]
#### Is your feature request related to a problem? Please describe.

In many use-cases, it would be valuable to understand the sample size needed to understand the statistical significance of a disparity (or lack thereof) between two groups. Calculators like: http://powerandsamplesize.com/Calculators/Compare-k-Proportions/1-Way-ANOVA-Pairwise

Offer a manual procedure and a formula. I believe that the calculations above can be useful in helping Fairlearn users understand if their sample size is big enough to draw conclusions about the comparison between the proportion values (i.e: FPR, TPR, FNR, TNR) of different groups

#### Describe the solution you'd like

A function that takes in the subgroups, the proportion values, and performs the powerandsample size calculations (linked above), to output the sample sizes need in each pairwise comparison. 

#### Describe alternatives you've considered, if relevant

Using the power and samplesize calculator manually

**Comment by romanlutz** [comment]
This is very useful. A first step (that's not nearly as sophisticated, of course) towards understanding whether we have enough data is #737 , but I definitely agree that we should go beyond that.

**Comment by michaelamoako** [comment]
Agreed @hildeweerts . I think the same can be said of Error Bars too (tagged you in that issue)

**Comment by riedgar-ms** [comment]
Like with the error bars, I think that this is a case where we shouldn't be trying to do this within Fairlearn (for the reasons @hildeweerts  outlines), but we should be making sure that we support users applying the appropriate statistical tests themselves. Part of that would be some worked examples with _different_ statistical tests (along with a note as to why the particular statistical test is appropriate, without a digression into a full course on statistics).

**Comment by romanlutz** [comment]
What's a good way forward here?
@riedgar-ms for the error bars you presumably use another package to calculate the bounds, right? Can't we do the same for these tests?
